python/lsst/ip/isr/fringe.py

```diff
@@ -112,26 +112,25 @@ def run(self, exposure, fringes, seed=None):
         if not self.checkFilter(exposure):
             return
 
-        if self.config.pedestal:
-            self.removePedestal(fringes)
-
         if seed is None:
             seed = self.config.stats.rngSeedOffset
         rng = numpy.random.RandomState(seed=seed)
 
-        if hasattr(fringes, '__iter__'):
-            #multiple fringe frames (placeholder implementation)
-            positions = self.generatePositions(fringes[0], rng)
-            fluxes = numpy.ndarray([len(positions), len(fringes)])
-            for i, f in enumerate(fringes):
-                fluxes[:,i] = self.measureExposure(f, positions, title="Fringe frame")
-        else:
-            #single fringe frame
-            positions = self.generatePositions(fringes, rng)
-            fluxes = self.measureExposure(fringes, positions, title="Fringe frame")
-            fluxes = fluxes.reshape([len(positions), 1])
+        if not hasattr(fringes, '__iter__'):
             fringes = [fringes]
 
+        mask = exposure.getMaskedImage().getMask()
+        for fringe in fringes:
+            fringe.getMaskedImage().getMask().__ior__(mask)
+            if self.config.pedestal:
+                self.removePedestal(fringe)
+
+        # placeholder implementation for multiple fringe frames
```

Comment:
I realise you're only moving this comment around, rather than adding it, but: is there a JIRA ticket for replacing this implementation? If so, could you link to it please? If not, could you create one and then link to it?